src/js/strategies/CCPInitiationStrategyInterface.js

```diff
@@ -0,0 +1,35 @@
+
+export default class CCPInitiationStrategyInterface {
+    constructor() {
+        console.log("CCPInitiationStrategyInterface initialized");
+    }
+
+    getName(){}
```

Comment:
Any way we can ensure that all these method need to be overriden? Do we throw an expection in each of these methods saying functions needs to be overriden?

Reply:
No, we don't throw any expections

Reply:
We will log in the console, we don't throw exceptions